boilerplate/app/services/reactotron/reactotron-config.ts

```diff
@@ -7,13 +7,10 @@ export interface ReactotronConfig {
   useAsyncStorage?: boolean
   /** Should we clear Reactotron when load? */
   clearOnLoad?: boolean
-  /** Root state logging. */
-  state?: {
-    /** log the initial data that we put into the state on startup? */
-    initial?: boolean
-    /** log snapshot changes. */
-    snapshots?: boolean
-  }
+  /** log the initial data that we put into the state on startup? */
+  logInitialState?: boolean
+  /** log snapshot changes. */
+  logSnapshots?: boolean
```

Comment:
I de-nested these, as it's easier to merge the config that way and there wasn't really a good reason to have them nested.